test/kotlin/integration/StatFlushIntegrationTest.kt

```diff
@@ -44,14 +44,29 @@ class StatFlushIntegrationTest {
     }
   }
 
+  @Test
+  fun `multipe stat sinks configured`() {
+    val countDownLatch = CountDownLatch(1)
+    engine = EngineBuilder()
+      .addLogLevel(LogLevel.DEBUG)
+      // Really high flush interval so it won't trigger during test execution.
+      .addStatsFlushSeconds(100)
+      .addStatSinks(listOf(statsdSinkConfig(8125), statsdSinkConfig(5000)))
+      .addGrpcStatsDomain("example.com")
+      .setOnEngineRunning { countDownLatch.countDown() }
+      .build()
+
+    assertThat(countDownLatch.await(30, TimeUnit.SECONDS)).isTrue()
+  }
+
   @Test
   fun `flush flushes to stats sink`() {
     val countDownLatch = CountDownLatch(1)
     engine = EngineBuilder()
       .addLogLevel(LogLevel.DEBUG)
-      .addStatsDPort(8125)
       // Really high flush interval so it won't trigger during test execution.
       .addStatsFlushSeconds(100)
+      .addStatSinks(listOf(statsdSinkConfig(8125), statsdSinkConfig(5000)))
```

Comment:
Could we update the assertion/expectation in this test so that it awaits multiple stats uploads?

Reply:
Yeah let me give that a go